python/chronos/src/bigdl/chronos/data/tsdataset.py

```diff
@@ -36,12 +37,17 @@
 
 
 class TSDataset:
-    def __init__(self, data, **schema):
+    def __init__(self, data, repair=True, **schema):
         '''
         TSDataset is an abstract of time series dataset.
         Cascade call is supported for most of the transform methods.
         '''
         self.df = data
+        # detect low-quality data and automatic repair (optional)
+        flag, self.df = quality_check_timeseries_dataframe(df=self.df,
```

Comment:
where do you want to use this `flag`?

Reply:
Actually I am not sure. There are two possible ways:
one is utilize this flag to propose different message for user like this:
```python
if flag is True:
  loggings.warn("Now that there is no low quality data here.")
elif flag is False:
  loggings.warn("There are still some low quality data.")

```
Another way is just omit this flag
```python
_, self.df = quality_check_timeseries_dataframe(df=self.df,
```

Reply:
I have no idea, but giving logs via `flag` seems like a good option.

Reply:
we could just omit it since complete warning and repairing information has been provided.

Reply:
> we could just omit it since complete warning and repairing information has been provided.

done